backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/ResetPasswordQueryService.kt

```diff
@@ -2,41 +2,28 @@ package app.ehrenamtskarte.backend.auth.webservice.schema
 
 import app.ehrenamtskarte.backend.auth.database.AdministratorEntity
 import app.ehrenamtskarte.backend.auth.database.Administrators
+import app.ehrenamtskarte.backend.exception.service.ProjectNotFoundException
+import app.ehrenamtskarte.backend.exception.webservice.exceptions.InvalidLinkException
+import app.ehrenamtskarte.backend.exception.webservice.exceptions.PasswordResetKeyExpiredException
 import app.ehrenamtskarte.backend.projects.database.ProjectEntity
 import app.ehrenamtskarte.backend.projects.database.Projects
 import com.expediagroup.graphql.generator.annotations.GraphQLDescription
-import graphql.GraphqlErrorException
 import org.jetbrains.exposed.sql.and
 import org.jetbrains.exposed.sql.not
 import org.jetbrains.exposed.sql.transactions.transaction
 import java.time.Instant
 
-class PasswordResetKeyExpiredException() : GraphqlErrorException(
-    newErrorException().extensions(
-        mapOf(
-            Pair("code", "PASSWORD_RESET_KEY_EXPIRED")
-        )
-    )
-)
-
-class InvalidPasswordResetLinkException() : GraphqlErrorException(
```

Comment:
this shouldn't be deleted since there is a separate template using this code

Reply:
Jep. You're right. That change was actually intended. When working with HTTP clients you also only work with a fixed amount of response codes, which you use for all endpoints. Depending on the endpoint the error code can have a different meaning. That's what I tried to do here. 
Of course the frontend code needs to be adjusted for this as well, which will be done in the follow up PR. 

I'll move that change in the other PR and revert here. This way I'll not accidentally introduce an error here.